### PR TITLE
psp-1370 display filterable properties on the map using geoserver.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: frontend/src/App.scss [hunk 1]
```diff
@@ -1,92 +1,99 @@
 @import './colors.scss';
-.error {
-  color: $danger-color;
-}
+.App {
```
Comment (devinleighsmith) [inline frontend/src/App.scss:2]: increase specificity of app selectors.

File: frontend/src/App.scss [hunk 2]
```diff
@@ -1,92 +1,99 @@
...
-//by default, highlight dropdown options blue
-.dropdown-item,
-.dropdown-header {
-  color: $text-color;
-  &:hover {
-    color: #fff;
-    background-color: $active-color;
+  .form-select,
+  .form-control {
+    border: 1px solid #606060;
```
Comment (devinleighsmith) [inline frontend/src/App.scss:87]: change default styling of select/input boxes.

File: frontend/src/_variables.scss [hunk 3]
```diff
@@ -2,5 +2,5 @@
 
 $header-height: 72px;
 $navbar-height: 45px;
-$mapfilter-height: 76px;
+$mapfilter-height: 60px;
```
Comment (devinleighsmith) [inline frontend/src/_variables.scss:5]: corresponds to the margin change on the property filter.

File: frontend/src/components/common/form/SearchButton.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ const SearchButton: React.FC<ButtonProps> = ({ ...props }) => {
       <Button
         id="search-button"
         type="submit"
-        className={props.className ?? 'bg-warning'}
+        className={props.className ?? 'primary'}
```
Comment (devinleighsmith) [inline frontend/src/components/common/form/SearchButton.tsx:17]: search button is blue not yellow.

File: frontend/src/components/maps/leaflet/InventoryLayer.tsx [hunk 5]
```diff
@@ -172,50 +169,53 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
...
-      bbox: tile.bbox,
-      address: filter?.address,
-      municipality: filter?.municipality,
-      pid: filter?.pid,
-      organizations: filter?.organizations,
-      classificationId: filter?.classificationId,
-      minLandArea: filter?.minLandArea,
-      maxLandArea: filter?.maxLandArea,
-      name: filter?.name,
+      STREET_ADDRESS_1: filter?.STREET_ADDRESS_1,
```
Comment (devinleighsmith) [inline frontend/src/components/maps/leaflet/InventoryLayer.tsx:172]: filter object in same format as CQL filter properties.

File: frontend/src/store/slices/properties/useProperties.ts [hunk 6]
```diff
@@ -25,34 +25,26 @@ export const useProperties = () => {
    */
   const fetchProperties = useCallback(
     async (propertyBounds: API.IPropertySearchParams | null) => {
-      if (
```
Comment (devinleighsmith) [inline frontend/src/store/slices/properties/useProperties.ts:28]: remove unused bounds.

**Review by asanchezr — APPROVED** [review]

**Pull request merged**